packages/dds/matrix/src/test/performanceTestUtilities.ts

```diff
@@ -1,116 +0,0 @@
-/*!
```

Comment:
Note for reviewers: all of this functionality was moved to `matrixBenchmarks` (and no longer exported), since everything that needs it now also lives in that module.